Source/Utils/WarpXUtil.H

```diff
@@ -265,13 +339,6 @@ int queryWithParser (const amrex::ParmParse& a_pp, char const * const str, doubl
  * \param[in] num_val number of input values to use (optional with arrays, default is
  * amrex::ParmParse::LAST for reading until the last input value)
  */
```

Comment:
This comment block refers to the code that's deleted just below, it seems.

Reply:
I believe the comment refers to the function above it, so I'll move it up.